src/applications/facility-locator/actions/index.js

```diff
@@ -105,6 +106,65 @@ export const fetchProviderDetail = id => async dispatch => {
   }
 };
 
+/**
+ * Handles all urgent care request (mashup)
+ * @param {Object} parameters from the search request
+ * @returns {{}} An Object response (locations/providers)
+ */
+const returnAllUrgentCare = async params => {
+  const { address, bounds, locationType, page, center, radius } = params;
+  const urgentCareVaData = await LocatorApi.searchWithBounds(
+    address,
+    bounds,
+    locationType,
+    Object.keys(urgentCareServices)[1], // VA UrgentCare
```

Comment:
We should not depend on the order of the keys here.

```suggestion
  urgentCareServices.UrgentCare // VA UrgentCare
```